library/src/main/java/com/bumptech/glide/load/resource/bitmap/TransformationUtils.java

```diff
@@ -181,11 +224,11 @@ public static Bitmap centerInside(@NonNull BitmapPool pool, @NonNull Bitmap inBi
 
   /**
    * Sets the alpha of the Bitmap we're going to re-use to the alpha of the Bitmap we're going to
-   * transform. This keeps {@link android.graphics.Bitmap#hasAlpha()}} consistent before and after
+   * transform. This keeps {@link Bitmap#hasAlpha()}} consistent before and after
```

Comment:
Ok, this was done by the auto-formatter. Is there a way to import the default rules that are used in the glide project?